pkgs/misc/arm-trusted-firmware/default.nix

```diff
@@ -16,7 +16,7 @@ let
     src = fetchFromGitHub {
       owner = "ARM-software";
       repo = "arm-trusted-firmware";
-      rev = "refs/tags/v${version}";
+      rev = "v${version}";
       sha256 = "03fjl5hy1bqlya6fg553bqz7jrvilzrzpbs87cv6jd04v8qrvry8";
```

Comment:
No hash change?

Reply:
Ah :-/

Reply:
Addressed in 84ff95a95ac7992f4adef5af173d017bed8acfdc